build/ci-build.yml

```diff
@@ -0,0 +1,117 @@
+name: $(date:yyyyMMdd)$(rev:.r)
+
+trigger:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - src/*
+
+pr:
+  paths:
+    include:
+      - src/*
+      - build/ci-build.yml
+
+variables:
+  - group: 'Build Configuration'
+  # Always use fixed version for .NET Core SDK
+  - name: 'DotNet.Sdk.Version'
+    value: '2.2.206'
```

Comment:
We are using 2.2.7 (right?), so this should be `2.2.402`

PS: We should move this a variable group given we need this in CI build + release build. If we do that, I'd create a group per Arcus project.

Reply:
Yep, updated.